tasks/DisableCpuFreqScaling.rb

```diff
@@ -0,0 +1,27 @@
+# For background on this read the following two links
+# https://wiki.gentoo.org/wiki/Power_management/Processor
+# https://wiki.archlinux.org/index.php/CPU_frequency_scaling
+
+class DisableCpuFreqScaling
+  include Genesis::Framework::Task
+
+  init do
+    log "Make sure acpi-cpufreq is enabled"
+    run_cmd('modprobe acpi-cpufreq')
+  end
+
+  run do
```

Comment:
I thought about this one for a while and was thinking that this might quickly turn into a mess if we want to make it portable in any way. Another option might just be to check for for common cpu frequency service and blow the task up if they are running with a message like "wtf are you doing running this in your live boot os!".